JustTweak/Classes/Protocols/TweakProvider+decryptionClosure.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  TweakProvider+DecryptionClosure.swift
+//  Copyright © 2021 Just Eat Takeaway. All rights reserved.
+//
+
+import Foundation
+
+extension TweakProvider {
+    public var decryptionClosure: ((Tweak) -> TweakValue)? {
+        get {
+            nil
+        }
+        set {}
```

Comment:
It's just to have a default value to keep backwards compatibility. So this way adding a `decryptionClosure` is not required. Which will keep the changes to an "addition" instead of a "breaking change" -> so a minor version change. And because extensions can't contain stored properties, had to do it this way with a getter and setter. More info can be found in Alberto's comment: https://github.com/justeat/JustTweak/pull/62#issuecomment-981523559